index.js

```diff
@@ -32,30 +32,29 @@ function qppfc_standup_list() {
   return arr.join(", ");
 }
 
-function hcqis_standup() {
+// iQIES FC standup rotation
+function iqies_standup() {
   var arr1 = [
-     "chris b", "chris l", "jesse", "james", "john", "sampat", "tom s"
+     "chris", "james", "john", "josh", "tim"
   ];
   var arr2 = [
-    "tom w", "andrew", "manoj"
+    "tom", "bryon"
   ];
   arr1 = shuffle(arr1); 
   var arr = arr1.concat(arr2);
   console.log(arr.join(","));
   return arr.join(", ");
 }
 
-function iqies_standup() {
+// iQIES FC Scrum of Scrum rotation
+function iqies_sos() {
   var arr1 = [
-     "chris l", "james", "john", "prabha", "tim"
-  ];
-  var arr2 = [
-    "tom w", "ryan", "bryon"
+     "chris", "james", "john", "josh", "tim"
   ];
   arr1 = shuffle(arr1); 
-  var arr = arr1.concat(arr2);
-  console.log(arr.join(","));
-  return arr.join(", ");
+  var arr = arr1[0];
```

Comment:
I don't use JavaScript that often. Would it be more stylish to use arr1.shift() rather than arr1[0]?

Reply:
@matthopson can you make a suggestion?  I am not a js wiz either.

Reply:
You can certainly use `shift` to grab that element, however it's destructive to the array, so if you expect the array to be mutated (that element is pulled from it and all the other elements shift their position), then go for it!

Reply:
Let's try it with shift()